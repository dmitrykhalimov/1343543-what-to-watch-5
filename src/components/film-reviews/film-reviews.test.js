import React from "react";
import renderer from "react-test-renderer";
import FilmReviews from "./film-reviews";

describe(`FilmReviews`, () => {
  it(`Should FilmReviews render correctly`, () => {
    const tree = renderer
      .create(
          <FilmReviews
            comments={[]}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
