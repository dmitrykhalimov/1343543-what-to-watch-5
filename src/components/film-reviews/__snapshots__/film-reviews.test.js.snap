// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmReviews Should FilmReviews render correctly 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  />
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
